lib/modules/datasource/github-runners/readme.md

```diff
@@ -1,11 +1,12 @@
-This datasource returns a list of all _stable_ runners that are hosted by GitHub.
-This datasource ignores beta releases.
+This datasource returns a list of all runners that are hosted by GitHub.
 The datasource is based on [GitHub's `runner-images` repository](https://github.com/actions/runner-images).
 
 Examples: `windows-2019` / `ubuntu-22.04` / `macos-13`
 
 ## Maintenance
 
-New _stable_ runner versions must be added to the datasource with a pull request.
-Unstable runners are tagged as `[beta]` in the readme of the [`runner-images` repository](https://github.com/actions/runner-images).
-Once a runner version becomes stable, the `[beta]` tag is removed and the suffix `latest` is added to its YAML label.
+New runner versions must be added to the datasource with a pull request.
+Unstable runners are tagged as `[beta]` in the readme of the [`runner-images` repository](https://github.com/actions/runner-images) and should have the `isStable:false` property.
+Deprecated runners are tagged as `[deprecated]` in the readme and should have the `isDeprecated:true` property.
+Once a runner version becomes stable, the `[beta]` tag is removed and the suffix `latest` is added to its YAML label and the `isStable:false` should be removed.
+Once a runner version becomes deprecated, the `isDeprecated:true` property should be added.
```

Comment:
After you apply my suggestion above, can you add new `h3` headings, and move the text to the correct place? Here's what I mean:

```
## Maintenance

### Adding a new beta version

### Marking a version as stable

### Deprecating a version
```

Reply:
can you please make another suggestion now i've merged yours 🤗